Sources/Client/Model/Channel+Requests.swift

```diff
@@ -0,0 +1,457 @@
+        
+        return Client.shared.request(endpoint: .markRead(self)) { (result: Result<EventResponse, ClientError>) in
+            completion(result.map({ $0.event }))
+        }
+    }
+    
+    /// Send an event.
+    /// - Parameters:
+    ///   - eventType: an event type.
+    ///   - completion: a completion block with `Event`.
+    @discardableResult
+    func send(eventType: EventType, _ completion: @escaping Client.Completion<Event>) -> URLSessionTask {
+        return Client.shared.request(endpoint: .sendEvent(eventType, self)) { (result: Result<EventResponse, ClientError>) in
+            Client.shared.logger?.log("🎫 \(eventType.rawValue)")
+            completion(result.map({ $0.event }))
+        }
+    }
+    
+    /// Delete a message.
+    /// - Parameters:
+    ///   - message: a message.
+    ///   - completion: a completion block with `MessageResponse`.
+    @discardableResult
+    func delete(message: Message, _ completion: @escaping Client.Completion<MessageResponse>) -> URLSessionTask {
+        return message.delete(completion)
+    }
+    
+    /// Flag a message.
+    /// - Parameters:
+    ///   - message: a message.
+    ///   - completion: a completion block with `FlagMessageResponse`.
+    @discardableResult
+    func flag(message: Message, _ completion: @escaping Client.Completion<FlagMessageResponse>) -> URLSessionTask {
+        guard config.flagsEnabled else {
+            completion(.success(.init(messageId: message.id, created: Date(), updated: Date())))
+            return .empty
+        }
+        
+        return message.flag(completion)
+    }
+    
+    /// Unflag a message.
+    /// - Parameters:
+    ///   - message: a message.
+    ///   - completion: a completion block with `FlagMessageResponse`.
+    @discardableResult
+    func unflag(message: Message, _ completion: @escaping Client.Completion<FlagMessageResponse>) -> URLSessionTask {
+        guard config.flagsEnabled else {
+            completion(.success(.init(messageId: message.id, created: Date(), updated: Date())))
+            return .empty
+        }
+        
+        return message.unflag(completion)
+    }
+    
+    // MARK: - Members
+    
+    /// Add a member to the channel.
+    /// - Parameters:
+    ///   - member: a member.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func add(_ member: Member, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        return add(Set([member]), completion)
+    }
+    
+    /// Add members to the channel.
+    /// - Parameters:
+    ///   - members: members.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func add(_ members: Set<Member>, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        var members = members
+        
+        self.members.forEach { existsMember in
```

Comment:
Instead of this, I suggest:
`let newMembers = members.subtracting(self.members)`

https://developer.apple.com/documentation/swift/set/1783322-subtracting